src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
```

Comment:
Checklist point 5 - the discount amount may change later, it would be better to create a variable here (and for the $50 amount)

Reply:
Fixed :)